opensrp-chw/src/main/java/org/smartregister/chw/interactor/FamilyProfileInteractor.java

```diff
@@ -47,13 +46,17 @@ public void run() {
     private boolean hasPhone(String familyBaseEntityId){
 
         final CommonPersonObject personObject = getCommonRepository(Utils.metadata().familyRegister.tableName).findByBaseEntityId(familyBaseEntityId);
-        final CommonPersonObjectClient client = new CommonPersonObjectClient(personObject.getCaseId(), personObject.getDetails(), "");
-        client.setColumnmaps(personObject.getColumnmaps());
+        if (personObject != null) {
```

Comment:
I would suggest we make these changes on the classes the relate to the malaria module